icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2041,6 +2046,52 @@ class U_I18N_API NumberFormatterSettings {
      */
     Derived scale(const Scale &scale) &&;
 
+    /**
+     * Specifies the usage of the unit ("length-person", "length-road",  
```

Comment:
Please update according to the email thread with @macchiati: we are using just "person" and "road" instead of "length-person" and "length-road".

Reply:
Done!